src/main/java/org/apache/commons/lang3/math/Fraction.java

```diff
@@ -31,8 +31,10 @@
  * based and thus suffers from various overflow issues. For a BigInteger based
  * equivalent, please see the Commons Math BigFraction class. </p>
  *
+ * @deprecated for fraction implementation from commons-numbers
```

Comment:
Please add a URL to the Javadoc for the favored implementation.

Reply:
Added reference to the favored implementation